site.yml

```diff
@@ -209,24 +209,30 @@
     - role: haproxy
       haproxy_type: swift
       tags: ['openstack', 'swift', 'control', 'haproxy', 'infra']
+      when: swift.enabled|bool
```

Comment:
So it would be very very strange to have hosts listed in the swiftnode group, but have swift disabled. This feels like an unnecessary conditional applied to each role here.